pykg2tbl/j2/templates/broader-terms.sparql

```diff
@@ -0,0 +1,18 @@
+{# Jinja Template --> Sparql Query
+ | Get any broader broader terms in a triple.
+ | variables in template:
+ |    - triple: triple to search from
```

Comment:
triple is bad name for this - as it reflects a uri (being only one third of a triple)

in the context of this use case that uri represents a `term` so I suggest using that